src/content/use-the-api/kubectl-gs/login.md

```diff
@@ -121,6 +123,46 @@ For subsequent logins, you can use that shorthand to easily select which install
 kubectl gs login example
 ```
 
+### Device authentication flow
+
+Run the `login` command with the `--device-auth` flag to authenticate using the device flow...
+
+```nohighlight
+kubectl gs login https://api.g8s.example.westeurope.azure.gigantic.io --device-auth
+```
+
+Watch the command's output. It will print the URL of the authentication endpoint:
+
+```nohighlight
```

Comment:
If this is meant for headless usage, then the only thing printed to stdout should be the URL, no human hints, so that a script or command line could easily consume it. Or JSON output, for example. If we keep the feature as-is, the flag could be called `--no-open-browser` since the remaining behavior seems to be the same.